src/pages/Profile/index.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { compose, pathOr, omit, mergeRight } from 'ramda';
+import { message, Form } from 'antd';
+
+import PerfilContainer from '../../containers/Profile';
+import { updateSenha } from '../../services/password';
+
+const PerfilDash = ({ username, typeAccount }) => {
+  const [editar, setEditar] = useState(false);
+  const [form] = Form.useForm();
+
+  const error = () => {
+    message.error('Os dados do usuário não foram atualizados');
+  };
+
+  const handleCancel = () => {
+    form.resetFields();
+    setEditar(false);
+  };
+  
+  const handleOnSubmit = async (formData) => {
+
+    const resposta = await updateSenha(
+      mergeRight(
+        omit(['confirmPassword'], formData), 
+        { username }
+      ));
+
+    if (resposta.status === 422) {
+      error();
+    }
+    if (resposta.status === 200) {
+      success();
+      setEditar(false);
+      form.resetFields();
+    }
```

Comment:
o back-end pode retornar outros status que não sejam 422, por exemplo 401 quando a senha estiver errada, neste caso talvez seja melhor fazer a verificação caso o status for 200.
```suggestion
    if (resposta.status === 200) {
      success();
      setEditar(false);
      form.resetFields();
    } else {
       error();
    }
```